src/TypeScript.UnitTests/TsConfig/TsConfigProviderTests.cs

```diff
@@ -141,6 +141,163 @@ public async Task GetConfigForFile_TsConfigContainsFileWithIllegalCharacters_Fil
+
+            var eslintBridgeClient = new Mock<ITypeScriptEslintBridgeClient>();
+            SetupEslintBridgeResponse(eslintBridgeClient, new Dictionary<string, TSConfigResponse>
+            {
+                {
+                    "config1", new TSConfigResponse
+                    {
+                        Error = "some error",
+                        Files = new[] {testedFileName} // should be ignored
+                    }
+                },
+                {"config2", new TSConfigResponse()}
+            });
+
+            var logger = new TestLogger();
+            var testSubject = CreateTestSubject(tsConfigsLocator.Object, eslintBridgeClient.Object, logger);
+            var result = await testSubject.GetConfigForFile(testedFileName, CancellationToken.None);
+            result.Should().BeNull();
+
+            eslintBridgeClient.Verify(x => x.TsConfigFiles("config1", CancellationToken.None), Times.Once);
+            eslintBridgeClient.Verify(x => x.TsConfigFiles("config2", CancellationToken.None), Times.Once);
+            eslintBridgeClient.VerifyNoOtherCalls();
+
+            logger.AssertPartialOutputStringExists("some error");
+        }
+
+        [TestMethod]
+        public async Task GetConfigForFile_FailedToProcessTsConfig_ParsingError_TsConfigIsSkipped()
+        {
+            const string testedFileName = "tested\\file";
+            var tsConfigsInSolution = new[] { "config1", "config2" };
+            var tsConfigsLocator = SetupTsConfigsLocator(testedFileName, tsConfigsInSolution);
+
+            var eslintBridgeClient = new Mock<ITypeScriptEslintBridgeClient>();
+            SetupEslintBridgeResponse(eslintBridgeClient, new Dictionary<string, TSConfigResponse>
+            {
+                {
+                    "config1", new TSConfigResponse
+                    {
+                        ParsingError = new ParsingError{Code = ParsingErrorCode.UNSUPPORTED_TYPESCRIPT, Message = "some message", Line = 55},
+                        Files = new[] {testedFileName} // should be ignored
+                    }
+                },
+                {"config2", new TSConfigResponse()}
+            });
+
+            var logger = new TestLogger();
+            var testSubject = CreateTestSubject(tsConfigsLocator.Object, eslintBridgeClient.Object, logger);
+            var result = await testSubject.GetConfigForFile(testedFileName, CancellationToken.None);
+            result.Should().BeNull();
+
+            eslintBridgeClient.Verify(x => x.TsConfigFiles("config1", CancellationToken.None), Times.Once);
+            eslintBridgeClient.Verify(x => x.TsConfigFiles("config2", CancellationToken.None), Times.Once);
+            eslintBridgeClient.VerifyNoOtherCalls();
+
+            logger.AssertPartialOutputStringExists("some message");
+            logger.AssertPartialOutputStringExists(ParsingErrorCode.UNSUPPORTED_TYPESCRIPT.ToString());
```

Comment:
`AssertPartialOutputStringExists` takes a paramarray of strings, so you could check the message, code and line are all logged in a single assertion.
That would also check that they occur in the _same_ message.